src/watchlist/TickTradeSparkline.js

```diff
@@ -8,18 +8,20 @@ const ypos = (data, val) => ((val - min(data)) / (max(data) - min(data)) * 120);
 const history = data => data.map(x => x.quote);
 
 const TickTradeSparkline = (props) => {
-	const height = 100;
-	const width = 344;
+	const height = props.height || 100;
+	const width = props.width || 344;
 	const h = history(props.history);
 	const y = (height - ypos(h, h[h.length - 1])) || height;
 
+	const widthPerTick = 18;
+
 	return (
 		<div className="tick-sparklines">
 			<Sparklines
 				width={width}
 				height={height}
 				data={h}
-				limit={20}
+				limit={Math.round(width / widthPerTick)}
```

Comment:
opps, this is an accidental commit, but as screen gets bigger, dont we want to show more data?